networks/suzuka/suzuka-client/src/tests/mod.rs

```diff
@@ -0,0 +1,138 @@
+use anyhow::{Context, Result};
+use crate::{
+    coin_client::CoinClient,
+    rest_client::{Client, FaucetClient},
+    types::LocalAccount,
+};
+use once_cell::sync::Lazy;
+use std::str::FromStr;
+use url::Url;
+
+static MONZA_CONFIG : Lazy<movement_execution_util::config::Config> = Lazy::new(|| {
+    movement_execution_util::config::Config::try_from_env().context("Failed to create the config").unwrap()
+});
+
+// :!:>section_1c
+static NODE_URL: Lazy<Url> = Lazy::new(|| {
+
+    Url::from_str(
+       format!("http://{}", MONZA_CONFIG.monza_config.aptos_rest_listen_url.as_str()).as_str()
+    ).unwrap()
+    
+});
+
+static FAUCET_URL: Lazy<Url> = Lazy::new(|| {
+    
+    Url::from_str(
+        format!("http://{}", MONZA_CONFIG.monza_config.aptos_faucet_listen_url.as_str()).as_str()
+    ).unwrap()
+
+});
+// <:!:section_1c
+
+#[tokio::test]
+async fn test_example_interaction() -> Result<()> {
+    // :!:>section_1a
+    let rest_client = Client::new(NODE_URL.clone());
```

Comment:
Can we drop the comments `//!:>section_1a` they seem overally verbose

Reply:
Perhaps, I just copied this from the monza crate.

Reply:
It's because this is also a an example interaction copied from the Aptos upstream.

Reply:
Fine to leave for now IMO.